src/main/java/pl/task/parkingmeter/repository/VehicleService.java

```diff
@@ -0,0 +1,24 @@
+package pl.task.parkingmeter.repository;
+
+import org.springframework.stereotype.Repository;
+import pl.task.parkingmeter.entity.Vehicle;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
```

Comment:
to jest Serwis czy repozytorium?
po co nam taka przelotka 1:1

Reply:
service